test/Test/BaseDAO.hs

```diff
@@ -0,0 +1,297 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+module Test.BaseDAO
```

Comment:
Putting all tests into a single `Test.BaseDAO` module doesn't work in long term, soon we will have too many tests for one module. I propose to rename this module to `Test.BaseDAO.FA2` (since it targets FA2 functionality) and rename `test_BaseDAO` to `test_BaseDAO_FA2` or something like that.

Reply:
I've renamed this.